pipelines/build/common/build_base_file.groovy

```diff
@@ -75,6 +77,8 @@ class Builder implements Serializable {
                 SCM_REF: scmReference,
                 BUILD_ARGS: buildArgs,
                 NODE_LABEL: "${additionalNodeLabels}&&${platformConfig.os}&&${platformConfig.arch}",
+                CODEBUILD: platformConfig.codebuild as Boolean,
```

Comment:
I take it codebuild == true and no docker image provided is an invalid config? Worth doing a sanity check on that

Reply:
if codebuild == true and no docker image is provided it will already skip and continue as usual. e.g it won't break anything